docs/src/ref/trace_translators.md

```diff
@@ -78,7 +78,7 @@ Note that the transform DSL code does not specify what the two generative functi
 This information will be required for computing probabilities and probability densities of traces.
 We provide this information by constructing a **Trace Translator** that wraps the transform along with this transformation:
 ```julia
-translator = DeterministicTraceTranslator(p2, (), f)
+translator = DeterministicTraceTranslator(p2, (), choicemap(), f)
```

Comment:
DeterministicTraceTranslator accepts keyword arguments, and the bug was that I didn't include them in the original docs here. I want to keep this simple and avoid the choicemap() and the args I think.
```
translator = DeterministicTraceTranslator(p_new=p2, transform=f)
```
(then, we can rename `f` to `transform` as well below)

I can try to make these changes in this PR if you don't mind.

Reply:
Sounds good! I think we should just be consistent across all the TraceTranslator types -- `SymmetricTraceTranslator` currently uses `involution`, and all of the others use `f`, I believe.

Reply:
Oh, one thing to keep in mind though is that some code already uses `involution` or `f` as a keyword argument, and if we change the keyword argument now, that'll be a breaking change. I wonder if it's a good idea to just define `involution` and `f` as aliases of transform...? e.g. by adding constructor definitions like:

```julia
SymmetricTraceTranslator(;involution::TraceTransformDSL=error("No default for `involution`."), kwargs...) =
    SymmetricTraceTranslator(;transform=involution, kwargs...)
```

We can even throw in a deprecation warning.